packages/sanity/src/structure/components/pane/Pane.tsx

```diff
@@ -89,7 +89,7 @@ export const Pane = forwardRef(function Pane(
     ref.current = refValue
   }, [])
 
-  useEffect(() => {
+  useLayoutEffect(() => {
```

Comment:
This fixes a long standing quirk where clicking on a document sees a weird initial layout shift before the panel is mounted 😮‍💨